src/InteractiveWindow/VisualStudio/VsInteractiveWindowCommandFilter.cs

```diff
@@ -330,7 +330,7 @@ private int PreLanguageCommandFilterExec(ref Guid pguidCmdGroup, uint nCmdID, ui
                 switch ((CommandIds)nCmdID)
                 {
                     case CommandIds.AbortExecution: _window.Evaluator.AbortExecution(); return VSConstants.S_OK;
-                    case CommandIds.Reset: _window.Operations.ResetAsync(); return VSConstants.S_OK;
+                    case CommandIds.Reset: ((IInteractiveWindowOperations2)(_window.Operations)).ResetAsync(isFromSubmit : false); return VSConstants.S_OK;
```

Comment:
```_window``` appears to be passed in through the public API ```IInteractiveWindowEditorFactoryService.CreateTextView``` on ```VsInteractiveWindowEditorFactoryService```.  Is it safe to assume that it's an ```IInteractiveWindowOperations2```?